app/assets/javascripts/discourse/tests/helpers/d-select-assertions.js

```diff
@@ -0,0 +1,60 @@
+import QUnit from "qunit";
+import { query } from "discourse/tests/helpers/qunit-helpers";
```

Comment:
I've been slowly removing all uses of our `query` helper and I recommend ember's standard [`find`](https://github.com/emberjs/ember-test-helpers/blob/master/API.md#find) instead:

```js
import { find } from "@ember/test-helpers";
```